dashboard/src/shared/Secret.ts

```diff
@@ -1,54 +1,35 @@
 import { axiosWithAuth } from "./AxiosInstance";
-import { APIBase } from "./Kube";
-import { IK8sList, IOwnerReference, ISecret } from "./types";
+import { IK8sList, ISecret } from "./types";
+import * as url from "./url";
 
 export default class Secret {
-  public static async create(
```

Comment:
has this function gone somewhere else?

Reply:
ah, I see, this is not needed because the secrets are now being created in the backend.